README.md

```diff
@@ -68,7 +66,7 @@ the enclave once the host is re-started.
 - [Documentation](https://docs.conclave.net)
   - We recommend to read the [Concepts chapter](https://docs.conclave.net/enclaves.html) to learn more about confidential computing, enclaves, and Conclave's architecture
```

Comment:
Instead of "We recommend **to** read the", it should be "We recommend read**ing**".

Reply:
pushed in the change.